planning/development-strategy.md

```diff
@@ -42,6 +43,57 @@
    -  [ ] uppercase letter
    -  [ ] placed on the right side
 
+
+## `for: body` (_label_)
```

Comment:
I think it is better to create a label to the specific section. e.g "For our program". Because inside of the body we have header, main and footer.